pandas/io/json/normalize.py

```diff
@@ -224,6 +224,45 @@ def _pull_field(js, spec):
         sep = str(sep)
     meta_keys = [sep.join(val) for val in meta]
 
+    def _extract(obj, field, seen_meta, level):
+        """
+        Extract field from obj.
+        The result is stored in `records`, `lengths`, and `meta_vals`.
+
+        Parameters
+        ----------
+        obj : dict
```

Comment:
Is this a `dict` or a `JSON object`? The type and description are slightly in conflict

Reply:
It's a dict.